app/scripts/services/authManager.js

```diff
@@ -0,0 +1,43 @@
+/* global URI */
```

Comment:
bower angular-uri#0.1.0   ENORESTARGET No tag found that was able to satisfy 0.1.0